packages/app/src/phishing-banner.js

```diff
@@ -0,0 +1,151 @@
+// Test it here: https://codesandbox.io/s/hungry-colden-4zcny?file=/src/index.js
```

Comment:
this file isn't included anywhere in client code. It is included from the server.